README.md

```diff
@@ -1,5 +1,10 @@
 # Urban-Heat-Island
 Studying urban heat island effect in Austin, Texas using Google Earth Engine.
 
+To run main.py run ```uv run main.py -c``` in your terminal. This will prompt you enter a city name to generate a heat map for that urban center. To run the default map, run ```uv run main.py```, this will map the urban heat island of Austin, Texas as a default.
+
+## Running GOOGLE Earth Engine
+Using google earth engine requires an gmail, install ```geemap``` then, run this command in your virtual environment ```earthengine authenticate```. It will require you to follow the authentication process.
```

Comment:
Nice, good to keep this README updated from the start. 
Let me try this later today ...